src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -272,8 +272,14 @@ export class SVGViewer extends Component<SvgViewerProps, SvgViewerState> {
     });
   };
 
+  getCurrentAssetClasName = () => {
```

Comment:
Double ss missing, please ... adjust the other places in the code where thid happens 🙏

Reply:
@franrull thank you!

Reply:
@franrull done, please revisit